### ISSUE TITLE
SourceRank should incorporate contributor's guide

### ISSUE THREAD

**Issue by JonasT** [librariesio/libraries.io#1218]
The SourceRank should incorporate contributor's guide (by checking for common file names "contributing.txt", "contributing.md", ... with all lower/upper case variants) since this indicates more serious projects that actually make some effort to coordinate contributions.

**Comment by andrew** [comment]
@JonasT agreed, we're actually storing that data already as well for these types of files so it should be fairly easy to add in: https://github.com/andrew/libaries/blob/cac12b59140d949da6870cc6839d244fad7bea95/app/models/concerns/repo_manifests.rb#L31-L37
